client/extensions/wp-super-cache/accepted-filenames.jsx

```diff
@@ -18,216 +18,156 @@ import FormToggle from 'components/forms/form-toggle/compact';
+	renderToggle = ( fieldName, fieldLabel ) => {
+		const {
+			fields: { pages },
+			isRequesting,
+			isSaving,
+			translate,
+		} = this.props;
+
+		return (
+			<FormToggle
+				checked={ !! pages && !! pages[ fieldName ] }
+				disabled={ isRequesting || isSaving }
+				onChange={ this.handleToggle( fieldName ) }>
+				<span>
+					{ translate( '%(label)s)', { args: { label: fieldLabel } } ) }
+				</span>
+			</FormToggle>
+		);
+	}
+
+	handleToggle = ( fieldName ) => {
+		return () => {
+			const {
+				fields,
+				setAutosavingFieldValue,
+			} = this.props;
+			const groupName = 'pages';
+			const groupFields = fields[ groupName ] ? fields[ groupName ] : {};
+
+			if ( ! ( fieldName in groupFields ) ) {
+				return;
+			}
+
+			groupFields[ fieldName ] = ! groupFields[ fieldName ];
+			setAutosavingFieldValue( groupName, groupFields );
+		};
+	};
+
+	render() {
+		const {
+			fields,
+			handleChange,
+			handleSubmitForm,
+			isRequesting,
+			isSaving,
+			translate,
+		} = this.props;
+		const {
+			cache_acceptable_files,
+			cache_rejected_uri,
+		} = fields;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Accepted Filenames & Rejected URIs' ) }>
+					<Button
+						compact
+						primary
+						disabled={ isRequesting || isSaving }
+						onClick={ handleSubmitForm }>
+						{ isSaving
+							? translate( 'Saving…' )
+							: translate( 'Save Settings' )
+						}
+					</Button>
+				</SectionHeader>
+				<Card>
+					<form>
 						<FormLabel>
-							{ translate( 'Do not cache pages that contain the following strings:' ) }
+							{ translate( 'Do not cache these page types.' ) }
 						</FormLabel>
-						<FormTextarea
-							disabled={ isRequesting || isSaving }
-							onChange={ handleChange( 'cache_rejected_uri' ) }
-							value={ cache_rejected_uri && cache_rejected_uri.join( '\n' ) } />
-						<FormSettingExplanation>
-							{ translate(
-								'Add here strings (not a filename) that forces a page not to be cached. For example, ' +
-								'if your URLs include year and you dont want to cache last year posts, it’s enough ' +
-								'to specify the year, i.e. ’/2004/’. WP-Cache will search if that string is part ' +
-								'of the URI and if so, it will not cache that page.'
-							) }
-						</FormSettingExplanation>
-					</FormFieldset>
 
-					<FormFieldset>
-						<FormLabel>
-							{ translate( 'Whitelisted filenames:' ) }
-						</FormLabel>
-						<FormTextarea
-							disabled={ isRequesting || isSaving }
-							onChange={ handleChange( 'cache_acceptable_files' ) }
-							value={ cache_acceptable_files && cache_acceptable_files.join( '\n' ) } />
-						<FormSettingExplanation>
+						<FormSettingExplanation className="wp-super-cache__condition-settings-explanation">
 							{ translate(
-								'Add here those filenames that can be cached, even if they match one of the rejected ' +
-								'substring specified above.'
+								' See the {{a}}Conditional Tags{{/a}} ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Do not cache these page types. See the {{a}}Conditional Tags{{/a}} documentation for a complete discussion on each type.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).